manifest.yaml

```diff
@@ -32,7 +32,8 @@ arguments:
           - grpc
           - temporal
           - kafka
-    description: Any valid built-in service activities can be supplied here and they will be provided. The `service` flag must be set to true. Valid values for this are `http`, `grpc`, `temporal`, and `kafka`.
+          - python
+    description: Any valid built-in service activities can be supplied here and they will be provided. The `service` flag must be set to true. Valid values for this are `http`, `grpc`, `temporal`, `python` and `kafka`.
```

Comment:
Instead of putting `python` here, I think this should be scoped to the stencil-python module or w/e you're calling it for Outreach.

Otherwise, we're polluting this package.

Reply:
I'm not trying to hate on python, I meant that it shouldn't be inside of `stencil-golang` because it's scope isn't python applications.

Reply:
could you look at the usage? any other `working` choice?

Reply:
@deregtd any comment on this? We discussed adding `python` as yet another `serviceActivity` in [slack thread](https://outreach-hq.slack.com/archives/C01CAR6GVPD/p1678316708202279) already.  I've already abandoned the possibility of `python-grpc` since found one type is good enough when combining with either `http` or `grpc` for our decision makings.

Reply:
The key would be figuring out extensions points to stencil-golang to allow stencil-datascience to extend an automated GRPC interface on top of the basic golang stuff, and then insert a new container into stencil-base/devbase, etc.  stencil-golang should have no idea python exists, just as if we had stencil-python, it should have no idea stencil-golang exists, you'd be working with the base modules.  You're actually trying to do a specialization where you add MORE on top of stencil-golang, including adding a custom build of a python module, so you really want to compose both the idea of stencil-golang for the base service as well as a new second container on top of it.  So I'd suggest doing stencil-mlwrapper or stencil-wrappedpython or something like that.  stencil-python would be something like the analogy to stencil-golang, but for a python service as the only service getting deployed.  You're doing a hybrid, if that makes sense.

Reply:
Shouldn't that be module hooks within stencil-go, so that you can poke into it from other modules?

Reply:
It was difficult since I have to change a lot, after refactoring, it may be possible now. Still module hooks has less flexibility than the entire overwrite when we have new needs later. 
FYI: this is the motivation I mentioned in description for why I finally chose to do skipping instead of hacking into a given section. Now, it seems fine for the hook approach, but I don't know about the future. If you still have strong opinion here for using hooks after reading following and don't bother to get more PRs in the future about potential more customizations, I can go back to the original thinking.
```
skip Go traffic handler code generations for http and grpc when python is turned on, we'll make skeleton codes instead to proxy Python service behind. This helps to isolate our codes and minimize the needs of your efforts for review, instead of trying to hack by lots of if else statements in your codes from time to time when we try to keep evolving more.
```

Reply:
Also noticed that even when I switch to use Module hook to get rid of the usage of python `serviceActivities`, since `stencil-base` [still need it](https://github.com/getoutreach/stencil-base/pull/100/files#diff-885b271153664f100985178e148d28687d6220474d93e4526bf5f9e17c5263d9R60), we can't simply remove `python` from this repo since it will cause `stencil` to complain on validation error since we still need to feed `python` for `stencil-base`'s usage in the generated repo.

Reply:
Summary of today's meeting with @deregtd 
1. Due to the fact that rpc flag is already being used in `stencil-base`,  python is just a similar injection, We'll keep the flag adding in stencil-base and stencil-golang as MLP proposed now. DT will find ways to refactor later and MLP can participate the changes to make it still work for python community.
2. We'll use Module hook for the injection of new Go code in this repo instead of doing swapping, although it means more code changes now and maybe in the future.
I've pushed my changes on 2 accordingly. Please review and approve. @jaredallard if you still have concerns, let us know, if needed, I like to have another 1:1 meeting with you on an more Prague friendly time.